frontend/widgets/ProductRecommendations/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
The "CTA" button has missing top whitespace

![Screenshot 2019-07-23 at 13 13 07](https://user-images.githubusercontent.com/3393569/61707902-c32a6a00-ad4b-11e9-9523-21599717262c.png)

Reply:
done